frontend/dispatcher.lua

```diff
@@ -62,8 +62,10 @@ local settingsList = {
     reboot = { category="none", event="Reboot", title=_("Reboot the device"), device=true, condition=Device:canReboot(),},
     poweroff = { category="none", event="PowerOff", title=_("Power off"), device=true, condition=Device:canPowerOff(), separator=true,},
     show_menu = { category="none", event="ShowMenu", title=_("Show menu"), device=true,},
-    toggle_hold_corners = { category="none", event="IgnoreHoldCorners", title=_("Toggle hold corners"), device=true,},
-    toggle_rotation = { category="none", event="ToggleRotation", title=_("Toggle rotation"), device=true, separator=true,},
+    toggle_hold_corners = { category="none", event="IgnoreHoldCorners", title=_("Toggle hold corners"), device=true, separator=true,},
+    swap_rotation = { category="none", event="SwapRotation", title=_("Toggle orientation"), device=true,},
```

Comment:
can you change this back to `toggle_rotation` to mach the title and that way the event does not go away for users.